### PR TITLE
Create msg interface

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,12 +12,13 @@
...
   "devDependencies": {
     "@graphprotocol/graph-cli": "https://github.com/figment-networks/graph-cli.git#tendermint",
-    "@graphprotocol/graph-ts": "https://github.com/figment-networks/graph-ts.git#patiee/cosmos-txs"
+    "@graphprotocol/graph-ts": "https://github.com/figment-networks/graph-ts.git#cosmos"
   },
   "dependencies": {
     "as-proto": "^0.2.2",
     "babel-polyfill": "^6.26.0",
     "babel-register": "^6.26.0",
+    "lerna": "^4.0.0",
```
Comment (lavabyrd) [inline package.json:21]: Whats it used for in this situation? :)

**Comment by lavabyrd** [comment]
Is there a task ID for this? :)

File: package.json (hunk 1 above)
Comment (patiee) [reply package.json:21]: that import is missing, required for cosmos

File: src/cosmos.ts [hunk 2]
```diff
@@ -33,18 +34,22 @@ import {
...
   }
 }
 
-function saveTx(id: string, tx: cosmos.v1.Tx): void {
+function saveTx(id: string, tx: cosmos.v1.Tx, blockHash: Bytes, height: BigInt, hash: Bytes): void {
   const cTx = new Tx(id);
   cTx.authInfo = saveAuthInfo(id, tx.auth_info as cosmos.v1.AuthInfo);
+  cTx.blockHash = blockHash
   cTx.body = saveBody(id, tx.body as cosmos.v1.TxBody);
+  cTx.hash = hash
```
[the remark this answers is not in the document]
Comment (patiee) [reply src/cosmos.ts:46]: yes this is tx hash :)

**Pull request merged**